File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/EncryptedPassportElementFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.passport.EncryptedPassportElement;

public class EncryptedPassportElementFilterValidator extends FilterValidator<EncryptedPassportElement> {

    public EncryptedPassportElementFilterValidator(CustomValidator<EncryptedPassportElement> validator) {
        super(EncryptedPassportElement.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/EncryptedPassportElementFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.passport.EncryptedPassportElement
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class EncryptedPassportElementFilterValidator(validator: Validator<EncryptedPassportElement>) : FilterValidator<EncryptedPassportElement>(EncryptedPassportElement::class, validator)
